torch/_torch_docs.py

```diff
@@ -2966,6 +2991,30 @@ def merge_dicts(*dicts):
     torch.return_types.kthvalue(values=tensor([[4., 5., 6.]]), indices=tensor([[1, 1, 1]]))
 """.format(**single_dim_common))
 
+add_docstr(torch.lcm,
+           r"""
+lcm(input, other, out=None) -> LongTensor
+
+Computes the element-wise least common multiple (LCM).
```

Comment:
See above notes.

Reply:
In particular, what is lcm(a, 0)?

Should there be a test in test_torch.py explicitly for the gcd and lcm edge cases where one of the two inputs?

Reply:
Yeah, I think having explict tests is a good idea (I tested these edge cases against the numpy implementation manually). And, lcm(a, 0) = 0 (which is also the case in the numpy implementation).

Reply:
On second thought, doesn't [this line](https://github.com/pytorch/pytorch/blob/a79b41684742de3133b918b85f08b43f147098b3/test/test_torch.py#L19482) already account for this?

Reply:
You would think so because our test suite is so confusing, but I think that function is for the Math tests and your tests are in the method test section. I'm actually about to overhaul all of this so it's not super important where you test the extremal values. You can probably just add lcm to the math test portion, but some people have struggled to fit their functions in there. You would do that by adding to `torch_op_tests` on line 19546.